test/sourcemaps.js

```diff
@@ -26,8 +27,24 @@ describe('sourcemaps', function() {
       .readFileSync(path.join(__dirname, '/dist/index.map'))
       .toString();
     mapValidator(raw, map);
+    let mapObject = JSON.parse(map);
+    assert(
+      mapObject.sourceRoot ===
+        path.relative(b.options.outDir, b.options.rootDir),
```

Comment:
This test is incorrect on Windows because `sourceRoot` should always be a URL with forward slashes, but on Windows this will assert that it's a path with backslashes